src/subscribables/observable.js

```diff
@@ -63,8 +63,10 @@ ko.hasPrototype = function(instance, prototype) {
 };
 
 ko.isObservable = function (instance) {
-    return ko.hasPrototype(instance, ko.observable);
-}
+    //test if not constructor function;
+    return  ko.hasPrototype(instance, ko.observable) && (instance['fn'] === undefined);
```

Comment:
Knockout defines an `undefined` itself (and so doesn't use the global value). We can safely compare to `undefined`.